### DISCUSSION TITLE
kenapa alpha 5%?

### DISCUSSION THREAD

**Discussion by myibra** [teamalgoritma/community#54] [Q&A] [answered]
halo semuanya,
Saya masih penasaran terkait penentuan nilai alpha untuk melakukan uji hipotesis yang "biasanya" digunakan 5%. Pertanyaan saya adalah asal muasal nilai 5% yang dijadikan best practice penentuan alpha itu bagaimana ya? dan kondisi seperti apa kita bisa mengganti nilai alpha tersebut?

**Comment by Hertzspr** [comment]
Batas alpha 5% itu bisa dibilang hanya tradisi, spesifiknya bisa beda di domain masing-masing, dan bukan jadi penentu final. Lagipula ketika melewati batas arbitrary statistically significant belum tentu scientifically significant, begitu juga sebaliknya gagal melewati batas itu mungkin tidak statistically significant, tapi bisa jadi scientifically significant. Entahlah, saya juga noob, izin ikut menyimak.

**Comment by hwulanayu** [comment, marked as answer]
Hi @myibra, menambahkan jawaban dari @Hertzspr, benar bahwa nilai batas significance level atau alpha sebesar 5% merupakan nilai yang umum digunakan untuk uji hipotesis dalam berbagai penelitian. Mengutip Salkind (2007), 
> "In medical studies, it is common to set the
significance level to p < .01, whereas p < .05 is
common in the social sciences" (p. 890)."

Dari pernyataan tersebut, nilai alpha dapat berubah sesuai domain studi yang diteliti. Pertimbangan lain dalam menentukan nilai alpha adalah seberapa fatal ketika false positive atau type 1 error (peneliti melaporkan temuannya signifikan, padahal hanya terjadi secara kebetulan) terjadi. Nilai p 0.05 menunjukkan bahwa Anda bersedia menerima 5% sebagai batasan kemungkinan bahwa Anda salah ketika Anda menolak hipotesis nol.

Jika @myibra dan @Hertzspr ingin ekplor lebih lanjut mengenai significance level ini, dapat dimulai dari [Wikipedia: Statistical significance](https://en.wikipedia.org/wiki/Statistical_significance) dan membaca beberapa buku maupun paper yang disitasi pada halaman tersebut. Pada halaman tersebut juga diberikan sitasi mengenai kelemahan penggunaan significance level yang telah disinggung pada jawaban @Hertzspr. Happy learning :D

Referensi:
Salkind, N. J. (2007). Significance level. In Encyclopedia of measurement and statistics (Vol. 3, pp. 889–891). essay, Sage.
